packages/flutter/lib/src/services/text_input.dart

```diff
@@ -1125,6 +1125,10 @@ abstract class TextInputClient {
 
   /// Requests that the client remove the text placeholder.
   void removeTextPlaceholder() {}
+
+  /// Performs macOS specific selector from the NSStandardKeyBindingResponding
```

Comment:
If the app user has a custom selector defined in `DefaultKeyBinding.Dict`, the selector will be sent as-is to the framework right? So it's not necessarily from `NSStandardKeyBindingResponding`.

Reply:
Also in `DefaultKeyBinding.Dict` you can remap a key to a selector + parameters. An example:
```
"~h" = ("insertText:", "hello word", "moveBackward:");
```

Reply:
Also using the current primary focus seems to be a reasonable default implementation.

Reply:
I don't know if AppKit would delivery complete custom selectors to `doCommandBySelector:`. I can try it though.

`insertText:` is processed through `doCommandBySelector:` (and there is no argument other than selector there).

Reply:
Nevermind, custom selectors are delivered. I'll update the comment.

Reply:
Yeah I think it worked for me when I tried. But I didn't try a selector with a parameter list. The documentation I found didn't mention that either: https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/EventOverview/TextDefaultsBindings/TextDefaultsBindings.html

Reply:
Well, the signature of `doCommandBySelector:` only has one argument (the selector), so I don't see how parameters would work, but custom selectors certainly do.